app/graphql/types/outputs/ont/library_type.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Types
+  module Outputs
+    module Ont
+      # The type for Ont::Library objects.
+      class LibraryType < CommonOutputObject
+        field :name, String, null: false
+        field :plate_barcode, String, null: false
+        field :pool, Integer, null: false
+        field :pool_size, Integer, null: false
+        field :tag_set_name, String, null: true
+        field :tube_barcode, String, null: true
+      end
```

Comment:
As discussed `well_primary_grouping_direction` removed from the input type, but `pool_size` kept as it will remain available on the active model, and will probably vary with further implementations of Ont workflows